backend/routes/statDictionaries.js

```diff
@@ -50,10 +52,93 @@ let STAT_VARIANCES = {
   PTS: 0,
 };
 
-let POP_SIZE = 0;
+const OPPONENT_STYLE_TO_MYTEAM_STYLE = {
+  outsideOffenseScore: [
+    "outsideOffenseScore",
+    "defenseDisciplineScore",
+    "reboundingScore",
+    "offenseDisciplineScore",
+    "consistencyScore",
+    "insideOffenseScore",
+  ],
+  insideOffenseScore: [
+    "defenseDisciplineScore",
+    "outsideOffenseScore",
+    "insideOffenseScore",
+    "offenseDisciplineScore",
+    "reboundingScore",
+    "consistencyScore",
+  ],
+  offenseDisciplineScore: [
+    "outsideOffenseScore",
+    "insideOffenseScore",
+    "consistencyScore",
+    "offenseDisciplineScore",
+    "defenseDisciplineScore",
+    "reboundingScore",
+  ],
+  defenseDisciplineScore: [
+    "outsideOffenseScore",
+    "offenseDisciplineScore",
+    "reboundingScore",
+    "defenseDisciplineScore",
+    "consistencyScore",
+    "insideOffenseScore",
+  ],
+  consistencyScore: [
+    "offenseDisciplineScore",
+    "insideOffenseScore",
+    "consistencyScore",
+    "reboundingScore",
+    "outsideOffenseScore",
+    "defenseDisciplineScore",
+  ],
+  reboundingScore: [
+    "reboundingScore",
+    "offenseDisciplineScore",
+    "outsideOffenseScore",
+    "insideOffenseScore",
+    "consistencyScore",
+    "defenseDisciplineScore",
+  ],
+};
 
-function incrementPopSize() {
-  ++POP_SIZE;
-}
+const FEEDBACK_OPTIONS = {
```

Comment:
can you think of a different way to store these options? mainly try to avoid referring to a position in an array as this makes it ambiguous. (i.e. FEEDBACK_OPTIONS[score].description )